examples/connext_dds/custom_transport/c/FileTransport.c

```diff
@@ -1497,6 +1497,11 @@ static RTI_INT32 NDDS_Transport_FILE_create_sendresource_srEA(
         return 0;
     }
 
+    /* Check if dest_address_in is null */
```

Comment:
Move precondition check before you use the variable. I guess you'd need to do this check on line 1484 right?